docs/_data/meetups/qrm-usEast.yaml

```diff
@@ -0,0 +1,12 @@
+region: Quorum
+country: USA
+state: East Coast
```

Comment:
Technically we also include Canada. :)

Reply:
If I understand this setup correctly, we have to "pick one".